app/templates/views/broadcast/partials/area-map-postcodes.html

```diff
@@ -0,0 +1,142 @@
+<script src="{{ asset_url('javascripts/leaflet.js') }}"></script>
+<script>
+
+  var polygons = []
+
+  {% if broadcast_message.areas %}
+    {% for polygon in broadcast_message.simple_polygons_with_bleed.as_coordinate_pairs_lat_long %}
+      polygons.push(
+        L.polygon({{polygon}}, {
+          opacity: 1,
+          color: '#1d70b8',
+          fillColor: '#5694ca',
+          fillOpacity: 0.15,
+          weight: 2,
+          dashArray: [6, 7],
+          lineCap: 'butt'
+        })
+      );
+    {% endfor %}
+
+    {% for polygon in broadcast_message.polygons.as_coordinate_pairs_lat_long %}
+      polygons.push(
+        L.polygon({{polygon}}, {
+          color: '#0b0b0c',
+          fillColor: '#5694ca',
+          fillOpacity: 0.3,
+          weight: 2
+        })
+      );
+      var bounds = L.polygon({{polygon}}).getBounds();
+      var center = bounds.getCenter();
+
+    {% endfor %}
+  {% endif %}
+
+
+  var mapElement = document.getElementById('area-list-map');
+  var grandparent = mapElement.parentNode.parentNode;
+  var isInDetails = grandparent.className.indexOf('govuk-details') !== -1;
+  var areas = document.querySelectorAll('.area-list > .area-list-item');
+  var trimRegExp = /^[\s\uFEFF\xA0]+|[\s\uFEFF\xA0]+$/g;
```

Comment:
What are we using this for, it feels unnecessary (since the server side should be able to handle it anyway), either way I think it's unusual enough to warrant a comment.

Reply:
Aria-label now created in jinja and both aria-label and static aria-description are set within area-map.html, so this has been removed from partials.